### DISCUSSION TITLE
LEDの配線について

### DISCUSSION THREAD

**Discussion by cubnosuke** [Yowkees/keyball#102] [Q&A] [answered]
お世話になります。
keyball61について質問させてください。
右手ボール仕様の右基板のLED26のDINのパターンが剥がれて導通不良になりました。
LED25までは点灯しています。LED25のDINからLED26の DINへの配線の追加したところ、全てのLEDが光っているのですが、この対応で問題はないでしょうか。

**Comment by koron** [comment, marked as answer]
既にtwitterで解決されたようですが、LEDのDOUTとDINの関係は「ひとつ前の番号のDOUTを、DINに繋ぐ」となります。

Keyballで採用しているSK6812mini-eをはじめとした
NeoPixelと言われるLEDは個々のLEDを制御する信号を
数珠繋ぎ(ディージーチェーン)で伝達しています。

そのためLED26のDINには、LED25のDOUTを繋ぐ必要があります。

**Reply by cubnosuke** [reply]
ありがとうございます！
修正できました。
詳しい解説もありがとうございます。

**Comment by Yowkees** [comment]
お問い合わせに気づかず申し訳ありませんでした！
ツイッターの方にも回答させていただきましたが写真の青矢印２点をショートしていただければと思います。in inでつないでも一応光ると思いますが、LEDのアクションが入った時に1つズレると思いますので、out -> inでつないでください。
よろしくお願いいたします。
![DINDOUT](https://user-images.githubusercontent.com/84209772/159829513-d7d701df-f8be-4305-9199-99200a79840f.jpg)

**Reply by cubnosuke** [reply]
最初に違った方に問い合わせしてまして、失礼しました。
ご回答ありがとうございます。
お忙しいなか、お手数おかけしてすいませでした。
また、ショップを利用させて頂くと思いますのでその節はよろしくお願いします。

**Reply by Yowkees** [reply]
ありがとうございます🙏今後ともよろしくお願いいたしますm(_ _)m
